wizeline/gepp-superset-fork
# Patch to export all records from a query on csv

**What does this PR do?**
Patch `viz.py` file to export query results without limit on number of rows.

**How should this be manually tested?**
-Run `Superset`
-Open a dashboard
-Select `Export csv` on some chart (preferably one that is limit to show a fixed number of records) 
-It should download all data available for that chart
